src/librustdoc/html/static/main.js

```diff
@@ -2792,6 +2792,10 @@ function defocusSearchBar() {
         addClass(popup, "hidden");
         popup.id = "help";
 
+        var book_info = document.createElement("span");
+        book_info.innerHTML = "You can find more information on rustdoc in its \
+            <a href=\"https://doc.rust-lang.org/rustdoc/\">book</a>.";
+
```

Comment:
nit: this could be more consice
```suggestion
        book_info.innerHTML = "You can find more information in
            <a href=\"https://doc.rust-lang.org/rustdoc/\">the rustdoc book</a>.";

```